mob-validation.js

```diff
@@ -0,0 +1,52 @@
+const mobEmail = document.querySelector('#mob-email');
```

Comment:
- [ ] The error messages that you have created in your .js files are not showing near the submit button. Kindly fix it. Also  it would be more clearer if your submit button was descriptive, changing the inner HTML text from `See Project` to `Submit` or `Send` fixes that.

Your Design
![image](https://user-images.githubusercontent.com/81932478/169535232-1a6d1d32-b1eb-41b9-83e8-560eccc97dab.png)

Figma template
![image](https://user-images.githubusercontent.com/81932478/169535347-4b33ee10-371d-4fcb-a444-359e133965e0.png)